src/main/react-webapp/src/components/Home.js

```diff
@@ -13,7 +13,14 @@ const Home = () => {
     return(
         <div id="content">
             {/* Drop down emergancy menu */}
-
+            
+            <button style="padding: 15px 32px;  text-align: center;  text-decoration: none;
```

Comment:
You should put the styling in the css file but other than that it looks good.

Reply:
+1